source-code/sdk-js/src/adapter-sveltekit/examples/spa-static/src/routes/+page.ts

```diff
@@ -1,12 +1,12 @@
 import { browser } from "$app/environment"
-import { inlangSymbol } from "../inlang.js"
+import { getRuntimeFromData } from "@inlang/sdk-js/adapter-sveltekit/shared"
 import type { PageLoad } from "./$types.js"
 
 export const load = (async ({ parent }) => {
 	if (browser) {
 		const data = await parent()
 
-		const i = data[inlangSymbol]!.getInlangFunction()
+		const i = getRuntimeFromData(data).getInlangFunction()
```

Comment:
Why do we need to specifically call `.getInlangFunction()` here? Is it possible to offer `i` as a property:

`const {i} = getRuntimeFromData(data)`

Reply:
This is one of the refactorings I still need to finish ;)